mne/epochs.py

```diff
@@ -1599,6 +1600,25 @@ def __repr__(self):
         class_name = 'Epochs' if class_name == 'BaseEpochs' else class_name
         return '<%s | %s>' % (class_name, s)
 
+    def _repr_html_(self):
+        if self.baseline is None:
+            baseline = 'off'
+        else:
+            baseline = tuple(['None' if b is None else ('%g' % b)
+                              for b in self.baseline])
+            baseline = '%s: %s sec' % baseline
+            if self.baseline != _check_baseline(
+                    self.baseline, times=self.times, sfreq=self.info['sfreq'],
+                    on_baseline_outside_data='adjust'):
+                baseline = ' (baseline period was cropped after baseline\
+                    correction)'
+        events = ''
+        for k, v in sorted(self.event_id.items()):
```

Comment:
you need to be careful here as 

`event_id : int | list of int | dict | None`

Reply:
Alright! Should I also update [this part](https://github.com/mne-tools/mne-python/blob/e430158cee998776551efffbb343e4e42a45ae37/mne/epochs.py#L1594) of the `__repr__` method?